src/components/Footer.js

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import IconLink from "@components/typography/IconLink";
```

Comment:
1. Tutaj import tak samo trzeba zrobić `import { IconLink } from "@components/typography";`
2. IMO `typography` to nie jest odpowiednie miejsce dla IconLink. Lepiej stworzyć osobny podkatalog `link` i tam trzymać wszystkie linki. Typography powinien służyć tylko dla czystych, ostylowanych tekstów